### DISCUSSION TITLE
Testing Printer Status Simulation with Virtual ZPL Printer.

### DISCUSSION THREAD

**Discussion by Not-Jukkapan** [porrey/Virtual-ZPL-Printer#43] [General]
Hi, 
I'm currently using your Virtual ZPL Printer for a project and I was wondering if it's possible to simulate and test printer statuses, like checking if the printer is ready, busy, or has an error. Does your app support this kind of functionality?

**Comment by porrey** [comment]
This is not supported. The printer is designed mainly for testing basic communications and labels formats.
